perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_PAPER = "it_IT.UTF-8",
	LC_ADDRESS = "it_IT.UTF-8",
	LC_MONETARY = "it_IT.UTF-8",
	LC_NUMERIC = "it_IT.UTF-8",
	LC_TELEPHONE = "it_IT.UTF-8",
	LC_IDENTIFICATION = "it_IT.UTF-8",
	LC_MEASUREMENT = "it_IT.UTF-8",
	LC_TIME = "it_IT.UTF-8",
	LC_NAME = "it_IT.UTF-8",
	LANG = "en_GB.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("en_GB.UTF-8").
2018-04-14 06:51:52,049 - MainThread - SensorNodeFactory - INFO - skipping unknown node type
2018-04-14 06:51:52,051 - MainThread - SensorNodeFactory - INFO - cc2538-bsl.py  -p /dev/ttyUSB1 -a 0x00202000
Opening port /dev/ttyUSB1, baud 500000 address 0x202000
Connecting to target...
CC2538 PG2.0: 512KB Flash, 32KB SRAM, CCFG at 0x0027FFD4
Primary IEEE Address: 00:12:4B:00:11:F4:EB:44
2018-04-14 06:51:52,506 - MainThread - SensorNodeFactory - INFO - b''
2018-04-14 06:51:52,507 - MainThread - SensorNodeFactory - INFO - Found Zoul on /dev/ttyUSB1
2018-04-14 06:51:55,104 - MainThread - CoAPWrapper.2 - INFO - sudo ../../agent_modules/contiki/communication_wrappers/bin/tunslip6 -C -B 115200 -s /dev/ttyUSB1 fd00:c:2::1/64
<wishful_module_gitar.lib_arm_cortex_m.Zoul object at 0x73676270>
2018-04-14 06:51:56,206 - Thread-1   - CoAPWrapper.2 - INFO - ifconfig tun0 inet `hostname` mtu 1500 up
2018-04-14 06:51:56,231 - Thread-1   - CoAPWrapper.2 - INFO - ifconfig tun0 add fd00:c:2::1/64
2018-04-14 06:51:56,246 - Thread-1   - CoAPWrapper.2 - INFO - ifconfig tun0 add fe80::c:2:0:1/64
2018-04-14 06:51:56,256 - Thread-1   - CoAPWrapper.2 - INFO - ifconfig tun0
2018-04-14 06:51:56,258 - Thread-1   - CoAPWrapper.2 - INFO - 
2018-04-14 06:51:56,266 - Thread-1   - CoAPWrapper.2 - INFO - tun0      Link encap:UNSPEC  HWaddr 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00
2018-04-14 06:51:56,268 - Thread-1   - CoAPWrapper.2 - INFO - inet addr:127.0.0.1  P-t-P:127.0.0.1  Mask:255.255.255.255
2018-04-14 06:51:56,269 - Thread-1   - CoAPWrapper.2 - INFO - inet6 addr: fe80::c:2:0:1/64 Scope:Link
2018-04-14 06:51:56,270 - Thread-1   - CoAPWrapper.2 - INFO - inet6 addr: fd00:c:2::1/64 Scope:Global
2018-04-14 06:51:56,271 - Thread-1   - CoAPWrapper.2 - INFO - UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1500  Metric:1
2018-04-14 06:51:56,272 - Thread-1   - CoAPWrapper.2 - INFO - RX packets:0 errors:0 dropped:0 overruns:0 frame:0
2018-04-14 06:51:56,273 - Thread-1   - CoAPWrapper.2 - INFO - TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
2018-04-14 06:51:56,274 - Thread-1   - CoAPWrapper.2 - INFO - collisions:0 txqueuelen:500
2018-04-14 06:51:56,275 - Thread-1   - CoAPWrapper.2 - INFO - RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
2018-04-14 06:51:56,276 - Thread-1   - CoAPWrapper.2 - INFO - 
2018-04-14 06:51:56,830 - Thread-1   - CoAPWrapper.2 - INFO - Got configuration message of type C
2018-04-14 06:51:56,831 - Thread-1   - CoAPWrapper.2 - INFO - Setting prefix fd00:c:2::
2018-04-14 06:51:56,832 - Thread-1   - CoAPWrapper.2 - INFO - Removing wishful_control_address: fd00:c::2
2018-04-14 06:51:56,832 - Thread-1   - CoAPWrapper.2 - INFO - New wishful_control_address: fd00:c:2::2
2018-04-14 06:51:57,817 - Thread-1   - CoAPWrapper.2 - INFO - Control prefix set and apps started
<Task pending coro=<coap_event_server() running at /export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py:106>>
2018-04-14 06:52:08,794 - Thread-3   - pyre_discovery_module.main - INFO - Discovered Controller DL-tcp://10.1.1.1:8990, UL-tcp://10.1.1.1:8989
2018-04-14 06:52:25,218 - Thread-1   - CoAPWrapper.2 - INFO - PEEE 15
local monitor cp started - Name: ecoap_local_monitoring_program_cocoa_cc, Id: 1 - STARTED
2018-04-14 06:53:01,907 - Thread-1   - CoAPWrapper.2 - INFO - DEBUGG: RPL: Reset MRHOF
2018-04-14 06:53:16,685 - local_control_program - Contiki Base Connector - INFO - Attr coap_rx_success not found
2018-04-14 06:53:17,731 - Thread-1   - CoAPWrapper.2 - INFO - DEBUGG: RPL: Reset MRHOF
2018-04-14 06:53:18,746 - local_control_program - Contiki Base Connector - INFO - Attr coap_tx_failed not found
2018-04-14 06:53:20,802 - local_control_program - Contiki Base Connector - INFO - Attr burst_end_event not found
2018-04-14 06:53:22,864 - local_control_program - Contiki Base Connector - INFO - Attr burst_start_event not found
2018-04-14 06:53:24,920 - local_control_program - Contiki Base Connector - INFO - Attr rpl_rank not found
2018-04-14 06:53:25,927 - local_control_program - Contiki Base Connector - INFO - Attr rpl_rank not found
2018-04-14 06:53:26,934 - local_control_program - Contiki Base Connector - INFO - Attr coap_tx_success not found
2018-04-14 06:53:26,936 - local_control_program - Contiki Base Connector - INFO - Attr burst_end_event not found
2018-04-14 06:53:26,938 - local_control_program - Contiki Base Connector - INFO - Attr coap_rto not found
2018-04-14 06:53:26,939 - local_control_program - Contiki Base Connector - INFO - Attr coap_rx_success not found
2018-04-14 06:53:26,940 - local_control_program - Contiki Base Connector - INFO - Attr burst_start_event not found
2018-04-14 06:53:26,942 - local_control_program - Contiki Base Connector - INFO - Attr capacity not found
2018-04-14 06:53:26,943 - local_control_program - Contiki Base Connector - INFO - Attr IEEE802154_measurement_macStats not found
2018-04-14 06:53:26,944 - local_control_program - Contiki Base Connector - INFO - Attr coap_tx_failed not found
2018-04-14 06:53:27,952 - local_control_program - Contiki Base Connector - INFO - Attr app_stats not found
2018-04-14 06:53:27,954 - local_control_program - Contiki Base Connector - INFO - Attr coap_tx_success not found
2018-04-14 06:53:27,955 - local_control_program - Contiki Base Connector - INFO - Attr burst_end_event not found
2018-04-14 06:53:27,957 - local_control_program - Contiki Base Connector - INFO - Attr coap_rto not found
2018-04-14 06:53:27,958 - local_control_program - Contiki Base Connector - INFO - Attr coap_rx_success not found
2018-04-14 06:53:27,959 - local_control_program - Contiki Base Connector - INFO - Attr burst_start_event not found
2018-04-14 06:53:27,961 - local_control_program - Contiki Base Connector - INFO - Attr rpl_rank not found
2018-04-14 06:53:27,962 - local_control_program - Contiki Base Connector - INFO - Attr capacity not found
2018-04-14 06:53:27,963 - local_control_program - Contiki Base Connector - INFO - Attr rpl_preferred_parent not found
2018-04-14 06:53:27,965 - local_control_program - Contiki Base Connector - INFO - Attr ipv6_stats not found
2018-04-14 06:53:27,966 - local_control_program - Contiki Base Connector - INFO - Attr coap_tx_failed not found
2018-04-14 06:53:46,069 - Thread-1   - CoAPWrapper.2 - INFO - UDP client process startup initiated!
2018-04-14 06:53:46,073 - Thread-1   - CoAPWrapper.2 - INFO - UDP client process started!
2018-04-14 06:54:10,773 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 8
2018-04-14 06:54:10,867 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 9
2018-04-14 06:54:10,962 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 10
2018-04-14 06:54:11,058 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 11
2018-04-14 06:54:20,750 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 114
2018-04-14 06:54:20,843 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 115
2018-04-14 06:54:20,937 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 116
2018-04-14 06:54:21,031 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 117
2018-04-14 06:54:21,125 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 118
2018-04-14 06:54:21,218 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 119
2018-04-14 06:54:21,312 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 120
2018-04-14 06:54:21,406 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 121
2018-04-14 06:54:21,499 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 122
2018-04-14 06:54:21,593 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 123
2018-04-14 06:54:21,781 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 125
2018-04-14 06:54:21,875 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 126
2018-04-14 06:54:21,969 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 127
2018-04-14 06:54:22,062 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 128
2018-04-14 06:54:22,155 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 129
2018-04-14 06:54:22,249 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 130
2018-04-14 06:54:22,343 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 131
2018-04-14 06:54:22,437 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 132
2018-04-14 06:54:22,531 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 133
2018-04-14 06:54:22,625 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 134
2018-04-14 06:54:22,719 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 135
2018-04-14 06:54:22,813 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 136
2018-04-14 06:54:22,905 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 137
2018-04-14 06:54:22,999 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 138
2018-04-14 06:54:23,093 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 139
2018-04-14 06:54:23,187 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 140
2018-04-14 06:54:23,281 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 141
2018-04-14 06:54:23,375 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 142
2018-04-14 06:54:23,469 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 143
2018-04-14 06:54:23,562 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 144
2018-04-14 06:54:23,655 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 145
2018-04-14 06:54:23,749 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 146
2018-04-14 06:54:23,843 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 147
2018-04-14 06:54:23,937 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 148
2018-04-14 06:54:24,031 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 149
2018-04-14 06:54:24,125 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 150
2018-04-14 06:54:24,218 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 151
2018-04-14 06:54:24,312 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 152
2018-04-14 06:54:24,406 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 153
2018-04-14 06:54:24,500 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 154
2018-04-14 06:54:24,593 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 155
2018-04-14 06:54:24,687 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 156
2018-04-14 06:54:24,781 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 157
2018-04-14 06:54:24,874 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 158
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 274, in read_measurements
    resp_key_values[measurement.name] = measurement.datatype.read_bytes(response_message[line_ptr:])
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/lib_gitar.py", line 38, in read_bytes
    tpl = struct.unpack(tmp_fmt, buf)
struct.error: unpack requires a bytes object of length 2
2018-04-14 06:54:35,118 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 267
2018-04-14 06:54:44,772 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 370
2018-04-14 06:54:44,960 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 372
2018-04-14 06:54:53,530 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 463
2018-04-14 06:54:53,624 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 464
2018-04-14 06:54:53,718 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 465
2018-04-14 06:54:53,811 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 466
2018-04-14 06:54:53,905 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 467
2018-04-14 06:54:53,998 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 468
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 25, in set_net_parameter
    return node.set_parameters(param_list, param_key_values_dict)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 203, in set_parameters
    resp_key_values[param.name] = ControlDataType(self.platform.endianness_fmt, self.platform.get_data_type_format_by_name('INT8')).read_bytes(response_message[line_ptr:])
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/lib_gitar.py", line 38, in read_bytes
    tpl = struct.unpack(tmp_fmt, buf)
struct.error: unpack requires a bytes object of length 1
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 274, in read_measurements
    resp_key_values[measurement.name] = measurement.datatype.read_bytes(response_message[line_ptr:])
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/lib_gitar.py", line 38, in read_bytes
    tpl = struct.unpack(tmp_fmt, buf)
struct.error: unpack requires a bytes object of length 16
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 25, in set_net_parameter
    return node.set_parameters(param_list, param_key_values_dict)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 203, in set_parameters
    resp_key_values[param.name] = ControlDataType(self.platform.endianness_fmt, self.platform.get_data_type_format_by_name('INT8')).read_bytes(response_message[line_ptr:])
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/lib_gitar.py", line 38, in read_bytes
    tpl = struct.unpack(tmp_fmt, buf)
struct.error: unpack requires a bytes object of length 1
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 25, in set_net_parameter
    return node.set_parameters(param_list, param_key_values_dict)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 203, in set_parameters
    resp_key_values[param.name] = ControlDataType(self.platform.endianness_fmt, self.platform.get_data_type_format_by_name('INT8')).read_bytes(response_message[line_ptr:])
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/lib_gitar.py", line 38, in read_bytes
    tpl = struct.unpack(tmp_fmt, buf)
struct.error: unpack requires a bytes object of length 1
2018-04-14 06:57:36,434 - Thread-1   - CoAPWrapper.2 - INFO - OVERFLOW COAP: TRANS OVER
2018-04-14 06:57:37,398 - Dummy-2261 - coap - INFO - Response not recognized - sending RST.
2018-04-14 06:57:58,704 - Dummy-31   - coap - INFO - Response not recognized - sending RST.
2018-04-14 06:57:58,976 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 06:59:09,451 - Dummy-2295 - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 25, in set_net_parameter
    return node.set_parameters(param_list, param_key_values_dict)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 196, in set_parameters
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 06:59:10,457 - Dummy-2261 - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 25, in set_net_parameter
    return node.set_parameters(param_list, param_key_values_dict)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 196, in set_parameters
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 06:59:31,751 - Dummy-31   - coap.requester - INFO - Request timed out
2018-04-14 06:59:32,053 - Dummy-26   - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 267, in read_measurements
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 07:00:02,114 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 07:01:35,192 - Dummy-26   - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 267, in read_measurements
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 07:02:05,183 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5066
2018-04-14 07:02:05,270 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 07:02:05,280 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5067
2018-04-14 07:02:05,370 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5068
2018-04-14 07:02:05,464 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5069
2018-04-14 07:02:05,558 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5070
2018-04-14 07:02:05,651 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5071
2018-04-14 07:02:05,745 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5072
2018-04-14 07:02:10,245 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5120
2018-04-14 07:02:10,339 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5121
2018-04-14 07:02:10,433 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5122
2018-04-14 07:02:10,526 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5123
2018-04-14 07:02:10,620 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5124
2018-04-14 07:02:10,714 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5125
2018-04-14 07:02:10,807 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5126
2018-04-14 07:02:10,995 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5128
2018-04-14 07:02:11,089 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5129
2018-04-14 07:02:11,182 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5130
2018-04-14 07:02:11,276 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5131
2018-04-14 07:02:11,370 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5132
2018-04-14 07:02:11,464 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5133
2018-04-14 07:02:11,558 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5134
2018-04-14 07:02:11,652 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5135
2018-04-14 07:02:11,744 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5136
2018-04-14 07:02:11,932 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5138
2018-04-14 07:02:12,026 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5139
2018-04-14 07:02:12,120 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5140
2018-04-14 07:02:12,214 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5141
2018-04-14 07:02:12,308 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5142
2018-04-14 07:02:12,401 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5143
2018-04-14 07:02:12,495 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5144
2018-04-14 07:02:12,589 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5145
2018-04-14 07:02:12,682 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5146
2018-04-14 07:02:12,870 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5148
2018-04-14 07:02:12,963 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5149
2018-04-14 07:02:13,057 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5150
2018-04-14 07:02:13,151 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5151
2018-04-14 07:02:13,245 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5152
2018-04-14 07:02:13,339 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5153
2018-04-14 07:02:13,433 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5154
2018-04-14 07:02:13,527 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5155
2018-04-14 07:02:13,619 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 5156
2018-04-14 07:03:38,323 - Dummy-26   - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 267, in read_measurements
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 07:03:57,211 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6261
2018-04-14 07:03:57,304 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6262
2018-04-14 07:03:57,398 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6263
2018-04-14 07:03:57,492 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6264
2018-04-14 07:03:57,586 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6265
2018-04-14 07:03:57,680 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6266
2018-04-14 07:03:57,774 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 6267
2018-04-14 07:04:08,400 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 07:05:41,473 - Dummy-26   - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 267, in read_measurements
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 07:06:11,547 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 07:06:32,082 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7913
2018-04-14 07:06:32,176 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7914
2018-04-14 07:06:32,269 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7915
2018-04-14 07:06:32,363 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7916
2018-04-14 07:06:32,456 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7917
2018-04-14 07:06:32,550 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7918
2018-04-14 07:06:32,644 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7919
2018-04-14 07:06:32,832 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7921
2018-04-14 07:06:32,925 - Thread-1   - CoAPWrapper.2 - INFO - Buffer full, Req id 7922
2018-04-14 07:07:44,575 - Dummy-26   - coap.requester - INFO - Request timed out
Traceback (most recent call last):
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_contikibase/net_connector_module.py", line 56, in get_net_measurements_periodic_worker
    ret = node.read_measurements(measurement_list)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/wishful_module_gitar/rpc_node.py", line 267, in read_measurements
    response_message = self.com_wrapper.send(request_message)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 86, in send
    send_loop.run_until_complete(self.coap_send(send_loop, payload))
  File "/usr/lib/python3.4/asyncio/base_events.py", line 276, in run_until_complete
    return future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/asyncio/tasks.py", line 233, in _step
    result = coro.throw(exc)
  File "/export/home/testbed-user/wishful_raspberry/dev/lib/python3.4/site-packages/communication_wrappers/coap_wrapper.py", line 73, in coap_send
    response = yield from context.request(request).response
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
aiocoap.error.RequestTimedOut
2018-04-14 07:08:14,653 - Dummy-26   - coap - INFO - Response not recognized - sending RST.
2018-04-14 07:09:47,728 - Dummy-26   - coap.requester - INFO - Request timed out
bash: line 1:  5821 Terminated              python agent.py --config ecoap_external_advanced/ecoap_agent.yaml
